spec/automation/unit/method_validation/vm_reconfigure_task_complete_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe 'vmreconfiguretask_complete method' do
+  let(:ems)        { FactoryGirl.create(:ems_vmware, :tenant => Tenant.root_tenant) }
+  let(:miq_server) { EvmSpecHelper.local_miq_server }
+  let(:user)       { FactoryGirl.create(:user_with_email_and_group) }
+  let(:vm)         { FactoryGirl.create(:vm_vmware, :ems_id => ems.id, :evm_owner => user) }
+
+  it 'sends email' do
+    expect(GenericMailer).to receive(:deliver).with(:automation_notification,
+                                                    hash_including(:to   => user.email,
+                                                                   :from => "evmadmin@example.com"
+                                                                  )
+                                                   )
+    attrs = ["MiqServer::miq_server=#{miq_server.id}"]
+    attrs << "vm_id=#{vm.id}"
```

Comment:
@billfitzgerald0120 
Should this use the VmOrTemplate::vm=vm.id or is the method going to use the id. If you use the ::vm approach the object would be fetched by the engine and the method can access the object, else it has to do the $evm.vmdb to find the vm